client/my-sites/promote-post/components/credit-balance/index.tsx

```diff
@@ -0,0 +1,41 @@
+import './style.scss';
+import { memo, useState } from '@wordpress/element';
+import { translate } from 'i18n-calypso';
+import InfoPopover from 'calypso/components/info-popover';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import { BlazeCreditStatus, useBlazeCredits } from 'calypso/lib/promote-post';
+
+const CreditBalance = () => {
+	const [ balance ] = useState( 0 ); // Todo: Fetch balance from the API
```

Comment:
Modify this to simulate credit balance when testing (You also need to be an approved user, ping me in Slack if you need any help)